security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -602,12 +604,13 @@ When considering threats specific to an organization's cloud native capabilities
 - Denial of Service (DoS) resulting from a pod that does not have resource limits applied therefore consumes the entire node level CPU and memory, worker node is then lost
 - Elevation of privilege could happen if a pod is running with unrestricted or higher privileged pod security policy or by modifying the security context of a pod or a container
 
-Threat actors to consider for cloud native security are consistent with existing threat modeling:
+Threat actors to consider for cloud native security are consistent with existing threat modeling practices:
+
+- Malicious insider - An actor with malicious intent and with authorization to perform actions within the modeled system.
+- Uninformed insider - An actor with authorization to perform actions within the modeled system (assuming anyone can be duped).
```

Comment:
Is `spoofed` instead of `duped` better representation of the intent here?

Reply:
Great question. My initial reaction is that "duped" may be better as it leaves this open to any sort of accidental or confused action that a user might be compelled to take. To my mind, this is different to Spoofing which is usually more about assuming a specific role (implicit or explicit). That said; within the (clunky) STRIDE framework, it does fit into spoofing more than any other category. 

I think I'm comfortable with "duped" as "User was duped by spearphishing and clicked a CSRF link" is clearly the type of issue we're concerned with for an "Uniformed insider".

All that said, super happy to change it if you feel strongly about it.

Reply:
I think it is ok as is.  If you decide to change it, might consider adding more words to explain the two cases explained in this comment, something like "assuming anyone can be deceived or make an error with unintended results"